### PR TITLE
Week 3, Guess the Number

### PR DESCRIPTION
**Pull request by DanielCowman**

### CODE REVIEW

**Comment by halfaro1** [comment]
closing and reopening to force the automated tests

File: guessTheNumberUpgrade.py [hunk 1]
```diff
@@ -44,7 +45,15 @@ def askUserToGuess( times, secretNumber ):
 #   the 'userGuess' parameter is the answer entered by the user
 #   the 'userSecretNumber' parameter is the randomly generated number
 def evaluateAnswer( userGuess, userSecretNumber ):
-    
+    if userGuess < userSecretNumber:
+        print "Too low, try a higher number."
```
Comment (halfaro1) [inline guessTheNumberUpgrade.py]: @DanielCowman are you using Python 3? The print statement should work like this in Python 3: `print("Too low, try a higher number")`

**Comment by DanielCowman** [comment]
I'm using 2.7 at the moment, looks like I found my first gripe with 3 ;-). I've been spending too much time with Ruby which has very relaxed formatting standards. 

I'll add the parentheses and update to 3.

**Comment by halfaro1** [comment]
Thanks, add a comment to this pull request when you sync again using Python3, @DanielCowman

**Comment by DanielCowman** [comment]
All set, looks like it failed the test, I can add a failsafe so the user can't assign topLimit with a value of 1, let me know.

**Comment by halfaro1** [comment]
@DanielCowman I just realized you're in my late-start class, aren't you? This assignment wasn't assigned to your class. You're welcomed to continue working on it, but it's not required for a grade. Did you get to it by just navigating or GitHub, or did I leave a link to it in any lesson?

**Comment by DanielCowman** [comment]
Looks like I just went from the github page and not through blackboard therefore I did the wrong project, is there anyway I can submit the ticTacToe Project or is it too late?

**Comment by halfaro1** [comment]
@DanielCowman sure go ahead. I'm closing this pull request since you don't need to submit it.

**Pull request closed without merging**